cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -178,13 +177,14 @@ private Set<String> findVariables(String content) {
   }
 
   @Override
-  public void installPlugin(ToolPluginDescriptor plugin, Step step) {
+  public void installPlugin(ToolPluginDescriptor plugin, Step step, ProcessContext pc) {
 
     Path mavenPlugin = this.getToolPath().resolve("lib/ext/" + plugin.name() + ".jar");
     this.context.getFileAccess().download(plugin.url(), mavenPlugin);
 
     if (Files.exists(mavenPlugin)) {
       this.context.success("Successfully added {} to {}", plugin.name(), mavenPlugin.toString());
+      createPluginMarkerFile(plugin);
```

Comment:
same here

Reply:
Removed and replaced with boolean true return value.